dilinuxreader.txt

```diff
@@ -1,37 +1,37 @@
 [Section]
 Name = DiskInternals Linux Reader
-Version = 2.6
+Version = 2.7
 License = Freeware
 Description = A easy way to bridge between Windows/ReactOS and Ext2/Ext3/Ext4, HFS and ReiserFS file systems.
-Size = 21.2 MiB
+Size = 21.3 MiB
 Category = 12
 URLSite = http://www.diskinternals.com/linux-reader/
-URLDownload = http://www.diskinternals.com/download/Linux_Reader.exe
-SHA1 = 681c0edd288b2c995b2837e00fcf7a46c9146643
+URLDownload = https://web.archive.org/web/20180525220638/http://www.diskinternals.com/files/Linux_Reader.exe
```

Comment:
web archive need the magic `if_` string in it, otherwise this will redirect to a webpage.

Reply:
Corrected.